CHANGELOG.md

```diff
@@ -1,5 +1,196 @@
 # Changelog
 
+## 🎰 1.6.0
+
+- ### Features
+
+  - **_BREAKING CHANGE_: Require the `webpack_config` field in `wrangler.toml` to build with a custom configuration - [EverlastingBugstopper], [issue/296] [pull/847]**
+
+    Wrangler will no longer use a `webpack.config.js` at the root of your project to build your worker. If you would like to continue using a custom build configuration, you will need to specify the `webpack_config` field in your `wrangler.toml` like so:
+
+    ```toml
+    name = "my-worker"
+    workers_dev = true
+    account_id = "01234567890987654321234567890"
+    webpack_config = "webpack.config.js"
+    ```
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/847]: https://github.com/cloudflare/wrangler/pull/847
+    [issue/296]: https://github.com/cloudflare/wrangler/issues/296
+
+  - **API Token Support - [gabbifish], [issue/354] [pull/471]**
+
+    Wrangler can now be configured with API Tokens!
+
+    Don't worry, current configurations with an email address and a Global API Key will continue to work, but we highly recommend that you switch to API Tokens as they are a much more secure authentication method.
+
+    If you want to use API tokens, create an API token from the "Edit Cloudflare Workers" API token template [here](https://dash.cloudflare.com/profile/api-tokens), and copy/paste it in the `wrangler config` prompt. Alternatively, you can set the `CF_API_TOKEN` environment variable.
+
+    [gabbifish]: https://github.com/gabbifish
+    [pull/471]: https://github.com/cloudflare/wrangler/pull/471
+    [issue/354]: https://github.com/cloudflare/wrangler/issues/354
+  
+  - **Add the ability to preview without opening the browser - [EverlastingBugstopper], [issue/256] [pull/816]**
+
+    `wrangler preview` can now be called with a `--headless` flag that will not open the browser.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/816]: https://github.com/cloudflare/wrangler/pull/816
+    [issue/256]: https://github.com/cloudflare/wrangler/issues/256
+
+  - **Check for valid credentials when running `wrangler config` - [gabbifish], [issue/439] [pull/842]**
+
+    Wrangler now validates that credentials passed to `wrangler config` will work with Cloudflare's API.
```

Comment:
i think i'm pulling an @ashleygwilliams here but it could be useful to explain what would happen prior to this PR

Reply:
done :)